stdlib/public/CommandLineSupport/CommandLine.cpp

```diff
@@ -124,25 +124,27 @@ char **_swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
     if (szBufferSize == 0) {
       swift::fatalError(0,
                         "Fatal error: Could not retrieve commandline "
-                        "arguments: %u\n",
-                        GetLastError());
+                        "argument %d: %lu\n",
+                        i, static_cast<unsigned long>(GetLastError()));
```

Comment:
Nit: The cast should be unnecessary: `DWORD GetLastError();`.

Reply:
It's my pedantry: I'm printing an unsigned long, so I want to explicitly pass an unsigned long where printing happens.

Reply:
The pedantry can also go the other way, saying that the cast would suppress a warning if `DWORD` and `unsigned long` were _not_ the same. Will change. :)